src/frontend/planner_test/tests/testdata/output/over_window_function.yaml

```diff
@@ -62,7 +62,7 @@
     Failed to bind expression: lag(x)
 
     Caused by:
-      Invalid input syntax: Window function `lag` must have OVER clause
+      function lag(integer) does not exist, do you mean log
```

Comment:
Seems this error message is less precise?

Reply:
To me, the two error messages both look good. Actually I think it's equally possible that the user 1) forgets to write `OVER` clause when intended to do a window function call or 2) mistypes the normal scalar function `log` as `lag`.

Reply:
May need additional work to achieve msg like `function lag(integer) does not exist as a scalar function, do you mean log or do you forget the OVER clause` which is better. We can leave it to future.